packages/odyssey-react-mui/src/NativeSelect.tsx

```diff
@@ -126,6 +140,21 @@ const NativeSelect: ForwardRefWithType = forwardRef(
         uncontrolledValue: defaultValue,
       })
     );
+    const inputRef = useRef<HTMLSelectElement>(null);
+
+    useImperativeHandle(
+      inputFocusRef,
+      () => {
+        const element = inputRef.current;
+        return {
+          focus: () => {
+            element && element.focus();
```

Comment:
I'd like to have this match other places in the code:
```suggestion
            element?.focus();
```